cpu/stm32l0/include/periph_cpu.h

```diff
@@ -85,6 +85,17 @@ typedef struct {
     uint8_t ev_irqn;        /**< event IRQ */
 } i2c_conf_t;
 
+/**
+ * @brief   Define the number of available PM modes
+ */
+#define PM_NUM_MODES    (2U)
+
+/**
+ * @brief   Override the default initial PM blocker
+ * @todo   we block all modes per default, until PM is cleanly implemented
```

Comment:
The "stop mode" is working fine even if i need to do a clk_init in my RTC callback so that the UART continues printing characters. So this patch could be merged as "stop mode" is not activated by default.

The issue is that xtimer (based on TIM2) is stopped in stop mode, @aabadie do you thing it is possible to associate xtimer with low power timer (LPTIM1) from stm32l0 so that we still have a time base running in stop mode ?
For now, i need to use thread_sleep/thread_wakeup that is not the best way to schedule my tasks.